scraibe/diarisation.py

```diff
@@ -183,7 +184,7 @@ def _save_token(token):
     
     @classmethod
     def load_model(cls, 
-                    model: str = PYANNOTE_DEFAULT_CONFIG, 
+                    model: str = PYANNOTE_FALLBACK_CONFIG, 
```

Comment:
I do not like the idea of using a fallback as the default. This might be confusing.